db/schema.rb

```diff
@@ -45,4 +45,10 @@
     t.integer "subscriber_count"
   end
 
+  create_table "subscribers", force: :cascade do |t|
+    t.string "address"
```

Comment:
Might it be more explicit to call this email or email_address?

Reply:
I had it as `email_address` but it seemed redundant at this stage as we're in the email-alert-api and it seemed sufficiently implicit. Happy to change it if you disagree.

Reply:
ah yes true 👍